omnipaxos/src/sequence_paxos/follower.rs

```diff
@@ -217,8 +218,9 @@ where
     }
 
     pub(crate) fn handle_accept_stopsign(&mut self, acc_ss: AcceptStopSign) {
-        if self.internal_storage.get_promise() == acc_ss.n
-            && self.state == (Role::Follower, Phase::Accept)
+        if self.internal_storage.get_promise() != acc_ss.n {
```

Comment:
then in the next else if we need to check for `==` also

Reply:
If `acc.n` < the promise, then we want the leader to know its ballot is out-competed. The `acc.n` > the promise scenario really shouldn't happen. A leader will only send non-prepare messages to followers who have promised to it. This means the followers promise will be >= the ballot of any message sent by a leader. In case this does somehow happen we could send a PrepareReq just to be safe?

Reply:
You're right, larger than should not happen. And if it is equal, then it will be ignored if we're in recovery mode (in which we already sent the PrepareReq). Then perhaps we can change the if to `acc. n` < the promise since that is the only possible case.